swagger.yml

```diff
@@ -1566,6 +1566,11 @@ definitions:
           type: boolean
         review_url:
           type: string
+        site_capabilities:
```

Comment:
this is something I added to api side. serialized deploy object now has `site_capabilities`, which has several information regarding the site capabilities.
since this `site_capabilities` is not `key:value=string:string` hash (sometime value can be object), I... wasn't sure how to define this in swagger.
`asset_management` key inside of `site_capabilities` is only present (and be true) if asset management feature is on. otherwise, there will be no such key. I'm not sure if this definition is okay for that case also...
In the code, I'm also using `deploy.SiteCapabilities.AssetManagement` but I'm not sure how it's gonna be if there is no `asset_management` key.